src/test/java/org/datadog/jmxfetch/SimpleTestJavaApp2.java

```diff
@@ -0,0 +1,5 @@
+package org.datadog.jmxfetch;
+
+public class SimpleTestJavaApp2 extends SimpleTestJavaApp {
```

Comment:
Not sure we need this in a new file, I think it'll be better if you use an inner class in `TestApp.java`.

Reply:
Yes, the initial reason to use a new file was to have a simpler name to match. But it's it's probably fine to use a inner class too. Changed in my last commit.

Using a separate class:
```
class: org.datadog.jmxfetch.SimpleTestJavaApp2
```

Using a inner class:
```
class: org.datadog.jmxfetch.TestApp$1SimpleTestJavaAnotherApp
```